python/llm/test/benchmark/arc-perf-test-batch2.yaml

```diff
@@ -35,4 +35,9 @@ exclude:
   - 'bigcode/starcoder-15.5b-4bit:2048'

```

Comment:
I don't think it is necessary to change the yaml files for performance tests as all of these 5 arguments have default value for now?